README.md

```diff
@@ -54,12 +79,24 @@ In this phase everyone is thinking like a Designer.  Design thinking also relies
 
 ***Q1.5 Based on the feedback, how might you change your solution?***
 
+1. Implement geofence capability allowing user to establish area in which they would like the collar to function.
+1. Test it a lot, FDA approved.
+1. Reward system: if you avoid the activity for the whole specified time, you receive a massage
+1. Tracks your chemical balances, to make sure you aren't shocked when you're down in the dumps
```

Comment:
Tracking vitals might be a good suggestion here as well during the testing period, as you never know what such shocks could do.